Assets/Scripts/CamRoom/InteractableTest.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class InteractableTest : MonoBehaviour, IInteractable
+{
+    [SerializeField] public string id = string.Empty;
+
+    public string GetId()
+    {
+        return id;
+    }
+
+    public void Interact()
+    {
+        Rigidbody rb = GetComponent<Rigidbody>();
```

Comment:
Keep this variable as private member variable and GetComponent in Start function